### ISSUE TITLE
Dreamcast controller

### ISSUE THREAD

**Issue by yakaracolombia** [OpenRetroPad/OpenRetroPad#6]
you have a pic with dreamcast wired to db9 but there no are option in the code, can we mod the Saturn or genesis inputs to work with dreamcast or need special one

**Comment by moparisthebest** [comment]
Yes I have wired up a dreamcast controller but have not been able to make the code work yet, if anyone wants to have a go that would be great, otherwise I'll probably get back to it one day.
